tqdm/_utils.py

```diff
@@ -142,14 +142,20 @@ def __ge__(self, other):
 
 
 class SimpleTextIOWrapper(object):
-    """Provide just an encoding to the write method.
+    """Change only `.write()` of the wrapped object by encoding the passed
+    value and passing the result to the wrapped object's `.write()` method.
     """
 
+    # pylint: disable=too-few-public-methods
```

Comment:
Fair warning about this in case it bothers you.   I mean we could just save `write_bytes` as an attribute and use `if`'s or a method call at each place we write to handle the encoding instead of this wrapper class.

Reply:
I do think I'd like the function better generally but we end up passing the 'file' to static methods so it seems wrapping the object is more in the spirit.  But sure, the other stuff could probably be refactored if we really dislike this wrapper.

Reply:
such magic much looking the other way :D